be/src/service/internal_service.cpp

```diff
@@ -1097,9 +1208,8 @@ void PInternalServiceImpl::multiget_data(google::protobuf::RpcController* contro
                                          const PMultiGetRequest* request,
                                          PMultiGetResponse* response,
                                          google::protobuf::Closure* done) {
-    // Submit task to seperate ThreadPool for avoiding block bthread working pthread
-    ThreadPool* task_pool = StorageEngine::instance()->get_bg_multiget_threadpool();
-    Status submit_st = task_pool->submit_func([request, response, done, this]() {
+    DorisMetrics::instance()->multiget_data->increment(1);
+    _light_work_pool.offer([request, response, done, this]() {
```

Comment:
This is a heavy work, because it will open and read segment

Reply:
ok